src/components/modal/modal.component.js

```diff
@@ -5,65 +5,25 @@ import CSSTransitionGroup from 'react-transition-group/CSSTransitionGroup';
 import Events from '../../utils/helpers/events';
 import Browser from '../../utils/helpers/browser';
 import Portal from '../portal';
-import './modal.scss';
+import { StyledModal, StyledModalBackground } from './modal.style';
 
 const TIMEOUT = 500;
 /**
  * A Modal Component
  *
  * Abstract base class for all modals
- *
```

Comment:
I think we want to remove the `@` doc stuff but want to leave the top level class comment so that when we finish the adding the Typescript definitions the developers get some helpful information about how to use the component from intellisense etc